trompa_ce_client.egg-info/PKG-INFO

```diff
@@ -0,0 +1,46 @@
+Metadata-Version: 2.1
```

Comment:
looks like you mistakenly included this egg-info directory and the egg file (in dist/). It'd be good to rebase to remove these.

Reply:
Since I split the changes in two branches the .gitignore got replaced by the original one. Fixed it in the current state.